website/docs/migration.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_position: 9
+---
+
+# Migrating from MongoDB to FerretDB
+
+To ensure a smooth and successful migration from MongoDB, we offer several methods through which you can test your application with FerretDB.
+
+## Operation modes
+
+We offer multiple operation modes which help facilitate the testing of your application by enabling FerretDB to act as a proxy.
+For more details, refer to the [official documentation](/configuration/operation-modes.md).
+
+### Manual and automated testing with `diff-normal` mode
+
+For details on how to install FerretDB, refer to the [quickstart guide](/quickstart-guide/).
```

Comment:
That should be a relative path

@Fashander could you please update our writing guide to mention that links should be relative and provide an example of linking to the category?

Reply:
@Fashander We updated our guide, but did not added an example of linking to the category (directory)